trendmicro_deepsecurity/icon_trendmicro_deepsecurity/connection/connection.py

```diff
@@ -0,0 +1,54 @@
+import komand
+from .schema import ConnectionSchema, Input
+# Custom imports below
+
+import requests
+from requests.auth import HTTPBasicAuth
+
+from icon_trendmicro_deepsecurity.util.shared import tryJSON
+from icon_trendmicro_deepsecurity.util.shared import checkResponse
+
+
+class Connection(komand.Connection):
+
+    def __init__(self):
+        super(self.__class__, self).__init__(input=ConnectionSchema())
+
+    def connect(self, params):
```

Comment:
Referring to the comment I left above about creating a requests `Session`, this is where you would want to do it. 

```
self.session = Session()
self.headers = {}  # Dictionary of headers here
```

And then in your actions, utilize the session like `self.connection.session.post( ... )`

Reply:
done